ctr/src/lib.rs

```diff
@@ -86,28 +86,18 @@ where
 {
     /// Create new CTR mode instance using initialized block cipher.
     pub fn from_cipher(cipher: C, nonce: &GenericArray<u8, U16>) -> Self {
-        let mut n = [0u64; 2];
-
-        // TODO: replace with `u64::from_be_bytes` in libcore (1.32+)
-        unsafe {
-            ptr::copy_nonoverlapping(nonce.as_ptr(), n.as_mut_ptr() as *mut u8, 16);
-        }
+        let nonce = u128::from_be_bytes(nonce.as_slice().try_into().unwrap());
```

Comment:
Now that the MSRV has been upped to 1.32 the `TryFrom` wasn't made stable up until that point. Looks like `try_into()` is however used elsewhere, so should I relax the CI or...?